ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs

```diff
@@ -109,30 +109,30 @@ public int PeriodOfRecord
                 NotifyPropertyChanged(); 
             }
         }
-
         public ObservableCollection<FlowDoubleWrapper> AnalyticalFlows
         {
             get  {return _AnalyticalFlows; }
             set { _AnalyticalFlows = value; NotifyPropertyChanged();}
         }
-        public ObservableCollection<FlowDoubleWrapper> GraphicalFlows
-        {
-            get { return _GraphicalFlows; }
-            set { _GraphicalFlows = value; NotifyPropertyChanged(); }
-        }      
+        public TableWithPlotVM MyGraphicalVM { get; set; }
 
         #endregion
         #region Constructors
+        //This supports a fresh editor
         public AnalyticalFrequencyEditorVM(ComputeComponentVM defaultCurve,  EditorActionManager actionManager) : base(defaultCurve, actionManager)
         {
             _Mean = DefaultCurveData.LP3Mean;
             _StDev = DefaultCurveData.LP3StDev;
             _Skew = DefaultCurveData.LP3Skew;
             _POR = DefaultCurveData.LP3POR;
-
+            MyGraphicalVM = new TableWithPlotVM(new GraphicalVM("Graphical Flow Frequency","Probability","Flow"), true);
```

Comment:
does it matter that graphical could be stage or flow?

Reply:
It does. I'm currently not supporting stage. I want to get back into main as quickly as possible though. I've made a ticket to address this comment in a later pull request. https://github.com/HydrologicEngineeringCenter/HEC-FDA/issues/386